src/track.c

```diff
@@ -343,14 +402,15 @@ void tracking_channel_update(u8 channel)
         * NAP_TRACK_CARRIER_FREQ_UNITS_PER_HZ;
 
 #if 1
-      if (chan->int_ms != 1) {
+      if (chan->stage > 0 && chan->lock_detect.outo) {
         s32 I = (cs[1].I - chan->alias_detect.first_I) / (chan->int_ms - 1);
         s32 Q = (cs[1].Q - chan->alias_detect.first_Q) / (chan->int_ms - 1);
         float err = alias_detect_second(&chan->alias_detect, I, Q);
         if (fabs(err) > (250 / chan->int_ms)) {
```

Comment:
I'm not totally clear on where it comes from.. need to do a bit more work to understand / validate the alias detection / correction stuff.  That can be a separate PR/issue though, it was in master before this PR.